ignite/templates/module/create/files/base/x/{{moduleName}}/keeper/keeper.go.plush

```diff
@@ -61,7 +69,16 @@ func NewKeeper(
         capabilityScopedFn:  capabilityScopedFn,<% } %>
 		<%= for (dependency) in dependencies { %>
 		<%= toVariableName(dependency.KeeperName()) %>: <%= toVariableName(dependency.KeeperName()) %>,<% } %>
+		Params:  collections.NewItem(sb, types.ParamsKey, "params", codec.CollValue[types.Params](cdc)),
 	}
+	
+	schema, err := sb.Build()
+	if err != nil {
+		panic(err)
```

Comment:
Would be awesome if we could find a way to not panic inside a module

Reply:
If we don't panic here we'll panic in depinject and add extra error checks in app.go.
This isn't supposed to fail, so imho panicking is fine (did as we do in the sdk https://github.com/cosmos/cosmos-sdk/blob/1c9361ca4e6752cb790ccf7467ab8a4a74a26573/x/gov/keeper/keeper.go#L140)

Reply:
Yes!! I know, but seems so weird in Golang panic in some random files in the different levels of the code